sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/CryptographyClient.java

```diff
@@ -0,0 +1,227 @@
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.exception.ResourceNotFoundException;
+import com.azure.core.http.rest.Response;
+import com.azure.core.implementation.annotation.ReturnType;
+import com.azure.core.implementation.annotation.ServiceClient;
+import com.azure.core.implementation.annotation.ServiceMethod;
+import com.azure.security.keyvault.keys.KeyClientBuilder;
+import com.azure.security.keyvault.keys.models.Key;
+
+import static com.azure.core.implementation.util.FluxUtil.withContext;
+
+@ServiceClient(builder = KeyClientBuilder.class, isAsync = true, serviceInterfaces = CryptographyService.class)
+public final class CryptographyClient {
+    private CryptographyAsyncClient client;
+
+    /**
+     * Creates a KeyClient that uses {@code pipeline} to service requests
+     *
+     * @param client The {@link CryptographyAsyncClient} that the client routes its request through.
+     */
+    CryptographyClient(CryptographyAsyncClient client) {
+        this.client = client;
+    }
+
+    /**
+     * Gets the public part of the configured key. The get key operation is applicable to all key types and it requires the {@code keys/get} permission.
+     *
+     * @throws ResourceNotFoundException when the configured key doesn't exist in the key vault.
+     * @return A {@link Response} whose {@link Response#value() value} contains the requested {@link Key key}.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Response<Key> getKey() {
+        return withContext(context -> client.getKey(context)).block();
```

Comment:
In sync client, context is passed using an explicit "Context" argument.
[This](https://github.com/Azure/azure-sdk-for-java/blob/b513d739be4e53fea2f362ca7903576b7e6cb4bf/sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java#L117) is a good example.

Need that for all sync Client methods.

Reply:
Added Overloads with Context argument since I don't have 'WithResponse' variants for the API except for get Key.